tools/profiles/debug.json

```diff
@@ -5,7 +5,7 @@
                    "-fmessage-length=0", "-fno-exceptions",
                    "-ffunction-sections", "-fdata-sections", "-funsigned-char",
                    "-MMD", "-fno-delete-null-pointer-checks",
-                   "-fomit-frame-pointer", "-O0", "-g3", "-DMBED_DEBUG",
+                   "-fomit-frame-pointer", "-O1", "-g3", "-DMBED_DEBUG",
```

Comment:
-Og is what you're looking for.

Reply:
Thanks @pan- that makes more sense. I changed it to -Og. It only increased the ROM by about 4 kB vs. -O1.